### ISSUE TITLE
Add FAQ Schema for Basic page and/or Accordion

### ISSUE THREAD

**Issue by mjoneill** [uiowa/uiowa#1350]
- [ ] @mjoneill to provide reference json, explanation, answer questions, etc.
- [ ] Implement schema in cms

Reference documentation or FAQ schema here: 
* https://developers.google.com/search/docs/data-types/faqpage
* https://schema.org/FAQPage

Parent Story: #1290

**Comment by mjoneill** [comment]
Unless we were to create a new content type for FAQ questions and answers, I think this only becomes available as an option if/when accordions are used.  The schema is constructed by elements of the accordion...

Note: Not all accordions will be FAQs.

**Comment by mjoneill** [comment]
[faq-json.txt](https://github.com/uiowa/uiowa/files/4471198/faq-json.txt)

Reference file from Google.

@joewhitsitt

**Comment by joewhitsitt** [comment]
Given that accordions (like you said, not always FAQs) could be on any page (multiple sets even), the only way I can see effectively managing this is to provide a checkbox option on Accordion like "Is this a set of Frequently Asked Questions". If checked, we would programmatically turn the title/contents into schema and find a way to append this to the application/ld+json markup. Then this functionality would not be available in a non-accordion format such as a HTML definition list.

**Comment by mjoneill** [comment]
@joewhitsitt Yep.  I would be fine (for now) of having that limitation. If we learn through usage that this is inhibiting someone, we can look at it again.  

Conversely, if you know right now that we need to extend this scope, speak up.

**Comment by pyrello** [comment]
https://www.morpht.com/blog/boost-drupal-seo-faqpages
